main/getting-started/README.md

```diff
@@ -68,52 +41,77 @@ yarn build
 yarn link-cli /usr/local/bin/agoric
 ```
 
+
 ## Your First Agoric Dapp
 
 ![The Browser Environment](./assets/browser-environment.svg)
 
-### 1. Creating a project directory
-Let's install the default smart contracts and UI.
+In this guide, we're installing, running and interacting with the default smart contracts and UI.
+
+TODO: define/explain the various parts
+
+
+### Initialize the Dapp code
+
+For now, you have to **stay in the `agoric-sdk` directory**. This is a [known limitation](https://github.com/Agoric/agoric-sdk/issues/570) that will be fixed.
 
 ```sh
 # Initialize your dapp project.
 # Note: Change the `demo` name to something meaningful.
-# Remember to prefix this with `npx` if you chose that option.
 agoric init demo
+# `init` creates a folder with the name you specify and copies over all
+# the files you might need.
+
 # Go to its directory.
 cd demo
-```
-`init` creates a folder with the name you specify and copies over all
-the files you might need.
-
-### 2. Installing the dependencies
-Next, let's install the necessary JavaScript packages. This step might
-take a while.
 
-```sh
 # Install Javascript dependencies.
 agoric install
 ```
 
-### 3. Launching the Agoric server
-Next, let's start up the Agoric VM. This creates the "vats" in which
-our smart contract code will be run.
+
+### Start the local Agoric Node, wallet server and Dapp Server
+
+This is all done in one command
 
 ```sh
 # Run the local vat machine.
 agoric start
 ```
 
-### 4. Installing a contract
-Let's deploy our dapp on the Agoric VM.
+Keep this terminal open somewhere
+
+You can now open
+- [http://localhost:3000](http://localhost:3000) to see our demo DApp
```

Comment:
I haven't been able to see anything at this address

Reply:
I think :3000 is a typo, and the :8000 below is correct.